src/cart.ts

```diff
@@ -14,6 +14,7 @@ export interface CartItem {
 	id: string;
 	variantId: string | null;
 	quantity: number;
+	attributes: { key: string; value: string }[];
```

Comment:
Don't we usually all these properties?

Reply:
The AJAX API calls them `{ properties: { [key: string]: string } }`, the Storefront and Admin Graphql API calls them `{ attributes: { key: string, value: string }[] }`. Not sure which one to go with...